spack_support/packages/seissol-env/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
I guess this file is written by you, Ravil, based on some sample taken from Lawrence Livermore? So maybe alter the LICENCE header accordingly. But I am no expert on legal stuff.

Reply:
It is the standard license agreement enforced by spack. Basically, this line is automatically generated by spack when one creates an installation script. I checked other scripts from hdf5 and openmpi. It is the same there